cmd/lifecycle/analyzer.go

```diff
@@ -20,51 +21,71 @@ import (
 
 type analyzeCmd struct {
 	//flags: inputs
-	cacheDir      string
-	cacheImageTag string
-	groupPath     string
-	uid, gid      int
 	analyzeArgs
+	uid, gid int
+
+	cacheDir      string // Platform API < 0.7
+	cacheImageTag string // Platform API < 0.7
```

Comment:
We'll still use this in the newer API to check credentials, right?

Reply:
The Spec doesn't list cache as an input to analyzer anymore. It may make a return for additional validation, but it was decided against for this first pass. We made that decision [here](https://docs.google.com/document/d/13D7OZt8oDZ_8EAcCHuAcVv24wi-tkFVcHu_0J3Ptylk/edit#).